tests/unit/development/__init__.py

```diff
@@ -13,51 +13,81 @@ def has_trailing_space(s):
     "It's multiple lines long.",
     "",
     "Keyword arguments:",
+    "",
     "- children (a list of or a singular dash component, string or number; optional)",
-    "- optionalArray (list; optional): Description of optionalArray",
+    "",
+    "- id (string; optional)",
+    "",
+    "- aria-* (string; optional)",
+    "",
+    "- customArrayProp (list; optional)",
+    "",
+    "- customProp (optional)",
+    "",
+    "- data-* (string; optional)",
+    "",
+    "- in (string; optional)",
+    "",
+    "- optionalAny (boolean | number | string | dict | list; optional)",
+    "",
+    "- optionalArray (list; optional):",
+    "    Description of optionalArray.  ",
```

Comment:
Ah shoot, missed this before, but does this mean there is trailing whitespace in some of the docstrings? We should really avoid that. I see up above there's a `has_trailing_space` function, that gets used a couple of places in the tests like:
```py
assert not has_trailing_space(component_class_string)
```
Ideally any file we create in our code generators (or any string meant to be written to a file) should pass that test.

Reply:
Well, I needed the trailing white spaces ("  \n" )to make the spacing work right in both Markdown and in the console.  Do the trailing white spaces in comments cause a problem?